<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>655</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>231</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Partitioning Options</string>
   </property>
  </widget>
  <widget class="QLabel" name="LpartName">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Partition Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="TpartName">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Lmount">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Mount Point</string>
   </property>
  </widget>
  <widget class="QLabel" name="Lsize">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Size</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Tsize">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>140</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comfs">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>XFS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FAT</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ext4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HFS</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Lfs">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>FileSystem</string>
   </property>
  </widget>
  <widget class="QLabel" name="Lhd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>HardDisk</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comhd">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>sda</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sdb</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>sdc</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="add_part_Button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Partition</string>
   </property>
  </widget>
  <widget class="QPushButton" name="show_part_Button">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>360</y>
     <width>111</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Show Partitions</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Finish_Button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>430</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Finish</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_raid_Button">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Raid</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>401</width>
     <height>23</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="value">
    <number>90</number>
   </property>
  </widget>
  <widget class="QPushButton" name="Back_button">
   <property name="geometry">
    <rect>
     <x>442</x>
     <y>430</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>621</width>
     <height>101</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>619</width>
      <height>99</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
  <widget class="QLineEdit" name="TpartName_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>221</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
